shadows/framework/src/main/java/org/robolectric/shadows/ShadowView.java

```diff
@@ -236,13 +240,14 @@ protected void removeOnLayoutChangeListener(View.OnLayoutChangeListener onLayout
   @Implementation
   protected void draw(Canvas canvas) {
     Drawable background = realView.getBackground();
-    if (background != null) {
+    if (background != null && !useRealGraphics()) {
       Object shadowCanvas = Shadow.extract(canvas);
       // Check that Canvas is not a Mockito mock
       if (shadowCanvas instanceof ShadowCanvas) {
         ((ShadowCanvas) shadowCanvas).appendDescription("background:");
       }
     }
+    System.err.println("JULIA THE CANVAS: " + canvas);
```

Comment:
Hi @JuliaSullivanGoogle , is this log needed?

Reply:
This CL is still WiP.